app/controllers/api/avatars.php

```diff
@@ -313,9 +313,9 @@
     ->param('margin', 1, new Range(0, 10), 'Margin from edge. Pass an integer between 0 to 10. Defaults to 1.', true)
     ->param('download', false, new Boolean(true), 'Return resulting image with \'Content-Disposition: attachment \' headers for the browser to start downloading it. Pass 0 for no header, or 1 for otherwise. Default value is set to 0.', true)
     ->inject('response')
-    ->action(function (string $text, int $size, int $margin, bool $download, Response $response) {
+    ->action(function (string $text, int $size, int $margin, mixed $downloadLoose, Response $response) {
```

Comment:
```suggestion
    ->action(function (string $text, int $size, int $margin, mixed $download, Response $response) {
```

No point in adding the postfix, I was a bit confused by it, and how dependency injection even allow it to work. We can stay with original name and overwrite the value. True to all use cases in this PR.

Reply:
Addressed in all places